Dokan/Dokan/Domain/Sources/Domain/Domain.swift

```diff
@@ -0,0 +1,13 @@
+
+
+struct User :Decodable{
+    let id:Int?
+    let username, firstname, lastname, email: String?
+    let imageUrl: String?
+
+}
```

Comment:
```suggestion
struct User {
    let id: Int
    let email: String
    let username, firstname, lastname: String?
    let imageUrl: String?
}
```
- Domain Entities should not conform to decodable
- ID should never be an optional since each user will have an ID
- Move to a separate file Under Entities group